opentelemetry-proto/src/transform/metrics.rs

```diff
@@ -97,10 +97,12 @@ pub mod tonic {
                 Temporality::Cumulative => AggregationTemporality::Cumulative,
                 Temporality::Delta => AggregationTemporality::Delta,
                 other => {
-                    opentelemetry::global::handle_error(MetricsError::Other(format!(
-                        "Unknown temporality {:?}, using default instead.",
-                        other
-                    )));
+                    otel_warn!(
```

Comment:
i dont think end users will understand what this means and they won't have anything actionable to do with this either. Lets use Debug level?

Reply:
done